inspire/dojson/institutions/fields/bd90x99x.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+#
+# This file is part of DoJSON
+# Copyright (C) 2015 CERN.
+#
+# DoJSON is free software; you can redistribute it and/or
+# modify it under the terms of the Revised BSD License; see LICENSE
+# file for more details.
+
+"""MARC 21 model definition."""
+
+from dojson import utils
+
+from ..model import institutions
+
+
+@institutions.over('oai_pmh', '^909C0')
+@utils.for_each_value
+@utils.filter_values
+def oai_pmh(self, key, value):
+    """Local OAI-PMH record information."""
+    return {
+        'id': value.get('o'),
+        'set': value.get('p'),
+        'previous_set': value.get('q'),
+    }
+
+
+@institutions.over('creation_modification_date', '^961..')
+@utils.for_each_value
+@utils.filter_values
+def creation_modification_date(self, key, value):
+    """Original creation and modification date."""
+    return {
+        'modification_date': value.get('c'),
+        'creation_date': value.get('x'),
+    }
+
+
+@institutions.over('spires_sysno', '^909C0')
+@utils.for_each_value
+@utils.filter_values
+def spires_sysno(self, key, value):
+    """Old SPIRES number."""
+    return {
+        'spires_sysno': value.get('a')
+    }
+
+
+@institutions.over('collections', '^980..')
+@utils.for_each_value
+@utils.filter_values
+def collections(self, key, value):
+    """Collection this record belongs to."""
+    return {
+        'primary': value.get('a'),
+        'secondary': value.get('b'),
+        'deleted': value.get('c'),
+    }
+
+
+@institutions.over('deleted_recid', '^981..')
+@utils.for_each_value
+@utils.filter_values
+def deleted_recid(self, key, value):
+    """Collection this record belongs to."""
+    return {
+        'deleted_recid': value.get('a'),
+    }
```

Comment:
`deleted_recid()`, `collections()`, `spyrise_sysno()`, `creation_modification_date()`, `oai_pmh()` should be all part of central rules that could be imported by all configurations.